luigi/contrib/lsf.py

```diff
@@ -236,8 +235,8 @@ def _run_job(self):
         args += ["-W", str(self.runtime_flag)]
         if self.job_name_flag:
             args += ["-J", str(self.job_name_flag)]
-        args += ["-o", os.path.join(log_output[0:-1], "job.out")]
-        args += ["-e", os.path.join(log_output[0:-1], "job.err")]
+        args += ["-o", os.path.join("/".join(log_output[0:-1]), "job.out")]
```

Comment:
Sorry, why are you flipping back to "/".join()? That's not OS safe. Unless I'm missing something.

Reply:
I've modified the code to use the os.path.split and os.path.join functions.

The reason for using the "/".join() is that the os.path.join could only take in strings. Using the os.path.split should get around this issue.